src/Config/Converter.php

```diff
@@ -24,10 +24,12 @@ class Converter implements ConverterInterface
             'cron_group' => ['type' => 'string', 'default' => 'default'],
             'archive_old_files' => ['type' => 'bool', 'default' => false],
             'delete_old_files' => ['type' => 'bool', 'default' => false],
+            'archive_date_format' => ['type' => 'string', 'default' => 'dmYhis'],
+            'directory_permissions' => ['type' => 'int', 'default' => 0777],
```

Comment:
actually 0777 is not proper approach (I know it was like this in codebase and this was not correct), could you make it more restrictive (maybe this would be useful: https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/deployment/file-system-permissions.html)

Reply:
I was just maintaining backward compatibility. Should I change the default to 0755?

Reply:
@bartoszkubicki done

Reply:
Great, thanks